include/libdnf/base/transaction.hpp

```diff
@@ -48,7 +48,7 @@ class Transaction {
     get_resolve_logs();
 
     /// @return the transaction packages.
-    std::vector<libdnf::base::TransactionPackage> get_packages();
+    std::vector<libdnf::base::TransactionPackage> get_packages() const;
```

Comment:
I am curious why it returns a copy. I suggest to return a const reference to internally stored object.

Reply:
I added TODO.
I also renamed `get_packages` to `get_transaction_packages`.

Reply:
LGTM